features/cellular/framework/targets/UBLOX/N2XX/UBLOX_N2XX_CellularContext.cpp

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2019, Arm Limited and affiliates.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include "UBLOX_N2XX_CellularContext.h"
+#include "UBLOX_N2XX_CellularStack.h"
+#include "APN_db.h"
+
+namespace mbed {
+
+UBLOX_N2XX_CellularContext::UBLOX_N2XX_CellularContext(ATHandler &at, CellularDevice *device, const char *apn) :
+    AT_CellularContext(at, device, apn)
+{
+}
+
+UBLOX_N2XX_CellularContext::~UBLOX_N2XX_CellularContext()
+{
+}
+
+NetworkStack *UBLOX_N2XX_CellularContext::get_stack()
```

Comment:
So that user will know at compile time that `PPP `is not supported. Are suggesting that ?
```
#if NSAPI_PPP_AVAILABLE
#error PPP is configured but not supported.
#endif
```
than `#if !NSAPI_PPP_AVAILABLE` is not need over `NetworkStack *UBLOX_N2XX_CellularContext::get_stack()` function.

Reply:
On second thought it's good how it is... the reason being that I think that there is now no way to say that a specific modem driver does not support PPP at compile time for now.